RUNNING_LOCALLY.md

```diff
@@ -50,6 +50,24 @@ Change `register: country` to `register: school-type`.
 
 Update the schema in the same way, to `schema: school-type`.
 
+Update fields and registers location for environments different to beta in the same way.
+
+Fields locations per environment:
+
+Variable: `fieldsYamlLocation`
+Possible values:
+alpha: https://field.alpha.openregister.org/records.yaml
+beta: https://field.register.gov.uk/records.yaml
+discovery: https://field.discovery.openregister.org/records.yaml
+
+Registers locations per environment:
+
+Variable: `registersYamlLocation`
+Possible values:
+alpha: https://register.alpha.openregister.org/records.yaml
```

Comment:
Can you prefix these with `*` to make them bullet points? Right now they run in one line when rendered.